nixos/modules/services/misc/dwm-status.nix

```
@@ -30,27 +26,12 @@ in
         example = "dwm-status.override { enableAlsaUtils = false; }";
       };
 
-      order = lib.mkOption {
-        type = lib.types.listOf (
-          lib.types.enum [
-            "audio"
-            "backlight"
-            "battery"
-            "cpu_load"
-            "network"
-            "time"
-          ]
-        );
-        description = ''
-          List of enabled features in order.
-        '';
-      };
-
-      extraConfig = lib.mkOption {
-        type = lib.types.lines;
-        default = "";
+      settings = lib.mkOption {
+        inherit (format) type;
+        default = { };
```

An example would be nice especially for users migrating from the old config style:
```suggestion
        default = { };
        example = {
          order = [
            "battery"
            "cpu_load"
            "time"
          ];
          time = {
            format = "%F %a %r";
            update_seconds = true;
          };
        };
```